articles/virtual-machines/linux/multiple-nics.md

```diff
@@ -176,4 +177,75 @@ You can also use a `copyIndex()` to then append a number to a resource name, whi
 You can read a complete example of [creating multiple NICs using Resource Manager templates](../../virtual-network/virtual-network-deploy-multinic-arm-template.md).
 
 ## Next steps
-Review [Linux VM sizes](sizes.md) when trying to creating a VM with multiple NICs. Pay attention to the maximum number of NICs each VM size supports. 
\ No newline at end of file
+Review [Linux VM sizes](sizes.md) when trying to creating a VM with multiple NICs. Pay attention to the maximum number of NICs each VM size supports. 
+
+## Additional Steps After A Linux VM with two NICs Got Created
+**Overview**
+
+When creating multiple NICs for a Linux Guest-OS based VM
+it is required to create additional routing rules which allows to send and receive traffic
+belonging to a specific NIC only. Otherwise traffic belonging to eth1 can not be processed correct, 
+due to the defined default route.  
+
+
+**Solution**
+
+At first it is required to add two new routing tables to the file /etc/iproute2/rt_tables
+
+    echo "200 eth0-rt" >> /etc/iproute2/rt_tables
+    echo "201 eth1-rt" >> /etc/iproute2/rt_tables
+
+To make the change persistent and get them applied during the network stack activation it is required 
+to alter the */etc/sysconfig/network-scipts/ifcfg-eth0* and */etc/sysconfig/network-scipts/ifcfg-eth1* file.
+Alter the line *"NM_CONTROLLED=yes"* to *"NM_CONTROLLED=no"*.
+Without this step the additonal rules/routing we are going to add taking no effect.
+In case it is is prefered to use NetworkManger dispatcher scripts please read 'man NetworkManager' for more details
+ 
+Next step is to extend the routing tables. In order to make the next steps more visible let's assume we have the following setup in place
+
+*Routing*
+
+    default via 10.0.1.1 dev eth0 proto static metric 100
+    10.0.1.0/24 dev eth0 proto kernel scope link src 10.0.1.4 metric 100
+    10.0.1.0/24 dev eth1 proto kernel scope link src 10.0.1.5 metric 101
+    168.63.129.16 via 10.0.1.1 dev eth0 proto dhcp metric 100
+    169.254.169.254 via 10.0.1.1 dev eth0 proto dhcp metric 100
+    
+*Interfaces*
+
+    lo: inet 127.0.0.1/8 scope host lo
+    eth0: inet 10.0.1.4/24 brd 10.0.1.255 scope global eth0    
+    eth1: inet 10.0.1.5/24 brd 10.0.1.255 scope global eth1
+    
+    
+With the above information it is possible to create the following additional files as root

```

Comment:
I'd suggest showing the create file and contents as individual steps. Need to be very prescriptive. Can also use `sensible-editor`. Something like:

> Create a file named */etc/sysconfig/network-scripts/rule-eth0* with `sensible-editor`. Enter the following contents, then save the file:

```bash
from 10.0.1.4/32 table eth0-rt
to 10.0.1.4/32 table eth0-rt
```


Repeat for the other three files.